tutorials/11-probabilistic-pca/11_probabilistic-pca.jmd

```diff
@@ -353,43 +325,42 @@ You can find more details about this in e.g. [5].
 end;
 ```
 
-Instead of drawing samples of each entry in $w$ from a standard normal, this time we repeatedly draw $D$ samples from the D-dimensional MND, forming a $D \times D$ matrix $w$.
-This matrix is a function of $\alpha$ as the samples are drawn from the MND parameterized by $\alpha$.
+Instead of drawing samples of each entry in $W$ from a standard normal, this time we repeatedly draw $D$ samples from the $D$-dimensional MND, forming a $D \times D$ matrix $W$.
+This matrix is a function of $α$ as the samples are drawn from the MND parameterized by $α$.
 We also introduce a hyper-parameter $tau$ which is the precision in the sampling distribution.
+We also re-paramterise the sampling distribution, i.e. each dimension across all instances is a 60-dimensional multivariate normal distribution. Re-parameterisation can sometimes accelrate the sampling process.
 
-We instantiate the model and ask Turing to sample from it using NUTS sampler. The sample trajectories of $\alpha$ is plotted using the `plot` function from the packge `StatsPlots`.
+We instantiate the model and ask Turing to sample from it using NUTS sampler. The sample trajectories of $α$ is plotted using the `plot` function from the packge `StatsPlots`.
 
 ```julia
-ppca_ARD = pPCA_ARD(expression_matrix) # instantiate the probabilistic model
-chain_ppcaARD = sample(ppca_ARD, NUTS(), 500)
-
-StatsPlots.plot(group(chain_ppcaARD, :alpha))
+ppca_ARD = pPCA_ARD(mat_exp) # instantiate the probabilistic model
+chain_ppcaARD = sample(ppca_ARD, NUTS(), 500) # sampling
+StatsPlots.plot(group(chain_ppcaARD, :α))
 ```
 
-Again, we do some inference diagnostics. Here we look at the convergence of the chains for the $\alpha$ parameter. This parameter determines the relevance of individual components.
-We see that the chains have converged and the posterior of the alpha parameters is centered around much smaller values in two instances.
-In the following, we will use the mean of the small values to select the *relevant* dimensions (remember that, smaller values of $\alpha$ correspond to more important components.).
-We can clearly see from the values of $\alpha$ that there should be two dimensions (corresponding to $\bar{\alpha}_3=\bar{\alpha}_5≈0.05$) for this dataset.
+Again, we do some inference diagnostics. Here we look at the convergence of the chains for the $α$ parameter. This parameter determines the relevance of individual components.
+We see that the chains have converged and the posterior of the $α$ parameters is centered around much smaller values in two instances.
+In the following, we will use the mean of the small values to select the *relevant* dimensions (remember that, smaller values of $α$ correspond to more important components.).
```

Comment:
```suggestion
In the following, we will use the mean of the small values to select the *relevant* dimensions (remember that, smaller values of $\alpha$ correspond to more important components.).
```

Reply:
That's a really good manner of writing Latex - thanks very for correcting me.